config-service-impl/src/main/java/org/hypertrace/config/service/ConfigServiceGrpcImpl.java

```diff
@@ -270,8 +271,21 @@ private String getTenantId() {
                 ::asRuntimeException);
   }
 
-  // TODO : get the userId from the context
+  private String getUserEmail() {
+    return RequestContext.CURRENT
```

Comment:
1. should this be an exception or should we fall back? If we have no email available, then does it make sense to say "config service is not supported"? I don't believe so - it feels like a nice to have to me.
2. If we do throw, make sure to use contextual status exception builder. Also this isn't an argument, it's more of a failed assertion about expected system state. I'd call it INTERNAL.